cmd/fetcher/kodata/config.yaml

```diff
@@ -23,16 +23,19 @@ ingress:
   - s3:
       bucket: "gs-noauth://knative-releases"
       prefix: "net-istio/previous"
+    eventingService: istio
```

Comment:
Is there a reason why this is called `eventingService`? I was first confused with Eventing as we are in the `knative-serving` part of the yaml.

Reply:
IngressService or just ingress?